scrapers/houseofgord.yml

```
@@ -0,0 +1,44 @@
+name: HouseOfGord
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - houseofgord.com/d
+    scraper: sceneScraper 
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@id="detail_title"]/text()
+      Date:
+        selector: //div[@class="preview_feature_date"][1]/text()
```

video isnt always the first entry eg `https://www.houseofgord.com/d/anastasia--petra-pony-race`
```suggestion
        selector: //div[@class="preview_feature_date"][contains(text(),"Video")]/text()
```

---

There are 2 scenes that on the current the site don't have video, but I do have the video. So I combined our selectors.